examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>Shall we generate the random samples via pymc XD (https://www.pymc-labs.io/blog-posts/simulating-data-with-pymc/()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>This code is not actually executed, they are just code blocks within markdown. I did think of doing that, but the PyMC way of doing it is perhaps less clear to people. I am totally willing to add the PyMC way of doing it if we think it's important to present a coherent front, but it is a little more verbose.</p>